precompile/staking/src/lib.rs

```diff
@@ -97,11 +97,10 @@ where
 		Ok(true)
 	}
 
-	#[precompile::public("restake(uint256,uint256,uint8[])")]
+	#[precompile::public("restake(uint256,uint8[])")]
```

Comment:
@JayJay1024 Did we support restake on UI? Need to adapt if the answer is yes.

Reply:
> @JayJay1024 Did we support restake on UI? Need to adapt if the answer is yes.

The current UI does not support restaked. Do we need to support restake on KTON Staking App?

Reply:
No need.

Reply:
The current UI uses restake. I will adapt it.